src/attestedclaim/AttestedClaim.ts

```diff
@@ -43,24 +77,75 @@ export default class AttestedClaim implements IAttestedClaim {
     this.attestation = Attestation.fromObject(attestation)
   }
 
+  /**
+   * @description (ASYNC) Verifies whether this attested claim is valid. It is valid if:
+   * * the data is valid (see [[verifyData]]);
+   * and
+   * * the [[Attestation]] object associated to this attestated claim is valid (see [[Attestation.verify]], where the **chain** is queried).
+   *
+   * Upon presentation of an attested claim, a verifier would call this [[verify]] function.
+   * @returns A promise containing whether this attested claim is valid.
+   * @example
+   * ```javascript
+   * attestedClaim.verify().then(data => {
+   *    console.log('isVerified', data);
+   * });
+   * ```
+   */
   public async verify(): Promise<boolean> {
     if (!this.verifyData()) {
       Promise.resolve(false)
     }
     return this.attestation.verify()
   }
 
+  /**
+   * @description Verifies whether the data of this attested claim is valid. It is valid if:
+   * * the [[RequestForAttestation]] object associated with this attested claim has valid data (see [[RequestForAttestation.verifyData]]);
```

Comment:
Suggestion: Great, but I think this should be in the About like you did before. I thought the beginning was only meant to be a short description. 

Moving this after the example. Could be something we can discuss?